src/auto-complete.jsx

```diff
@@ -298,8 +298,8 @@ const AutoComplete = React.createClass({
   },
 
   handleKeyDown(event) {
-    switch (event.keyCode) {
-      case KeyCode.ENTER:
+    switch (keycode(event.keyCode)) {
```

Comment:
https://github.com/timoxley/keycode/issues/18 kinda scared me. But I don't see us using it anywhere anyway. I'll update the code thanks :+1: